src/data/ja.yml

```
@@ -36,12 +36,12 @@ home:
   start-creating: p5 Editor で作成を開始しましょう！
   p1xh1: こんにちは！
   p1x1: >-
-    p5.js は、アーティスト、デザイナー、教育者、入門者、その他すべての人々が
-    コーディングに触れられるようにすることを目的とした JavaScript ライブラリです！
-    p5.js は、ソフトウェアやその学習ツールにはすべての人々がアクセス可能であるべきという信念に基づき、無料かつオープンソースで提供されています。
+    p5.js は、アーティスト、デザイナー、教育者、入門者、その他すべての人々が コーディングに触れられるようにすることを目的とした JavaScript
```

[nits]
半角スペースは不要かなと思いました！

```suggestion
    p5.js は、アーティスト、デザイナー、教育者、入門者、その他すべての人々がコーディングに触れられるようにすることを目的とした JavaScript
```

---

おそらく、もともと改行だったところが、Github Actionの処理で行数調整が行われて、1行に統合された場所かもです。
英単語だったらスペース区切りで単語は区切られるけど・・・日本語だったらくっつけるべき・・・・
いやはや、これまた英語圏の考えかたの処理方式ですね・・・